MODULE.bazel

```diff
@@ -1,9 +1,13 @@
-# FTXUI Module.
-module(name = "ftxui", version = "6.1.8", compatibility_level = 6)
+# FTXUI module.
+module(
+    name = "ftxui",
+    version = "6.1.8",
+    compatibility_level = 6,
+)
 
-# Build deps.
+# Build dependencies.
 bazel_dep(name = "rules_cc", version = "0.1.1")
-bazel_dep(name = "platforms", version = "0.0.11")
+bazel_dep(name = "platforms", version = "0.0.10")
 
-# Test deps.
-bazel_dep(name = "googletest", version = "1.16.0.bcr.1")
+# Test dependencies.
+bazel_dep(name = "googletest", version = "1.14.0.bcr.1", dev_dependency = True)
```

Comment:
There was some discussion in bazelbuild/bazel about test-only deps as dev_dependencies: https://github.com/bazelbuild/bazel/issues/22187
iirc there's some ongoing work on optional dependencies, but I don't think that's in a state where it's usable yet.